generators/app/templates/README.md

```diff
@@ -10,88 +14,100 @@ This README is intended to help get you started. Definitely update and improve t
 
 ### Running <%= botName %> Locally
 
-You can test your hubot by running the following.
+You can test your hubot by running the following, however some plugins will not
+behave as expected unless their specific environment variables have been set.
```

Comment:
This is a good point, but the obvious question is... how do I know what environment variables are needed? And, how can I set the environment variables?